graph/src/data/query/error.rs

```diff
@@ -70,6 +86,17 @@ impl fmt::Display for QueryExecutionError {
             QueryExecutionError::MissingArgumentError(_, s) => {
                 write!(f, "No value provided for required argument: {}", s)
             }
+            QueryExecutionError::StoreQueryError(s) => {
+                write!(f, "Failed to resolve store query, error: {}", s)
```

Comment:
As far as the user/client is concerned, the store is an implementation detail. Let's make this 
```rust
ResolveEntitiesError(s) => {
  write!(f, "Failed to resolve entities: {}", s)
}
```